src/Datadog.Trace.ClrProfiler.Managed/Integrations/GraphQLIntegration.cs

```diff
@@ -149,8 +149,7 @@ public static object ExecuteAsync(object executionStrategy, object context, int
 
             try
             {
-                var graphQLAssembly = AppDomain.CurrentDomain.GetAssemblies()
-                                               .Single(a => a.GetName().Name.Equals(GraphQLAssemblyName));
+                var graphQLAssembly = executionStrategy.GetType().Assembly;
```

Comment:
I remember why this was implemented this way....the implementation type of this assembly may not exist in the `GraphQL` assembly. What we should really be doing here is getting the `Type` object that represents the interface `GraphQL.Execution.IExecutionStrategy` which definitely is in the `GraphQL` assembly.

```suggestion
                var executionStrategyInterfaceType = executionStrategy.GetInstrumentedInterface(GraphQLExecutionStrategyInterfaceName);
                var graphQLAssembly = executionStrategyInterfaceType.GetType().Assembly;
```

Reply:
Done